release-process.rst

```diff
@@ -170,7 +282,7 @@ on the public internals mailing list or in the security mailing (ideally using
 the bug tracker)
 
 Release managers selection
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+==========================
 
 About three months prior to the scheduled release of the first alpha release
 of the next minor or major version (around March 1st or shortly thereafter),
```

Comment:
```suggestion
of the next minor or major version (around April 1st or shortly thereafter),
```

We've moved Alpha 1 to a month later,
I think the correct thing to do would be to either take April 1st,
as the start date of the volunteer call, or change that to 4 months before the Alpha 1